app/controllers/conversations_controller.rb

```diff
@@ -109,8 +109,8 @@ def new
   def contacts_data
     current_user.contacts.sharing.joins(person: :profile)
       .pluck(*%w(contacts.id profiles.first_name profiles.last_name people.diaspora_handle))
-      .map {|contact_id, *name_attrs|
-        {value: contact_id, name: ERB::Util.h(Person.name_from_attrs(*name_attrs)) }
-      }
+      .map do|contact_id, *name_attrs|
+        { value: contact_id, name: ERB::Util.h(Person.name_from_attrs(*name_attrs)) }
+      end
```

Comment:
For the most part, I follow [Weirichs rule](https://web.archive.org/web/20140221124509/http://onestepback.org/index.cgi/Tech/Ruby/BraceVsDoEnd.rdoc) on `do/end` vs `{/}`, I dislike do end blocks in calls like `map`.